plugins/search/src/alpha.tsx

```diff
@@ -198,22 +198,24 @@ export const SearchPage = createPageExtension({
               <Grid item xs>
                 <SearchPagination />
                 <SearchResults>
-                  {({ results }) =>
-                    results.map((result, index) => {
-                      const { noTrack } = config;
-                      const { document, ...rest } = result;
-                      const SearchResultListItem =
-                        getResultItemComponent(result);
-                      return (
-                        <SearchResultListItem
-                          {...rest}
-                          key={index}
-                          result={document}
-                          noTrack={noTrack}
-                        />
-                      );
-                    })
-                  }
+                  {({ results }) => (
+                    <>
+                      {results.map((result, index) => {
```

Comment:
Can you maybe explain this change a bit more? A bit confused why the wrapping Fragment is required :/

Reply:
The `(resultSet: SearchResultSet) => JSX.Element` type doesn't allow for `JSX.Element[]`, so we wrap it in a fragment to return just a single element.

Might be worth changing `SearchResultProps` instead though, @camilaibs? 😁

Reply:
On second thought I'd rather make the type change in a followup, trying to keep actual changes in this one to a minimum 😅